src/libraries/System.Security.Cryptography.Algorithms/src/System/Security/Cryptography/MD5.cs

```diff
@@ -14,7 +14,10 @@ namespace System.Security.Cryptography
 
     public abstract class MD5 : HashAlgorithm
     {
-        protected MD5() { }
+        protected MD5()
+        {
+            HashSizeValue = 128;
```

Comment:
I wasn't sure what to do with `HashSizeValue = ...` in `Implementation (line 33). I left it for now, but it's probably redundant. Perhaps `Implementation` should `Debug.Assert(_hashProvider.HashSizeInBytes * 8 == HashSizeValue)`?